vendor/bat-native-ads/src/bat/ads/internal/features.cc

```diff
@@ -14,60 +14,59 @@ namespace ads {
 namespace features {
 
 namespace {
-const char kPageProbabilitiesStudyName[] = "PageProbabilitiesHistoryStudy";
-const int kDefaultHistorySize = 5;
+const char kActiveStudyName[] = "PageProbabilitiesHistoryStudy";
+const int kDefaultPageProbabilityHistorySize = 5;
 }  // namespace
 
-// Controls behaviour of the contextual ad matching mechanism, e.g. by
-// adjusting the number of page classifications used to infer user interest.
-const base::Feature kContextualAdsControl { "ContextualAdsControl",
-    base::FEATURE_DISABLED_BY_DEFAULT };
+// TODO(Moritz Haller): Rename feature in seed
+const base::Feature kPageClassifier { "ContextualAdsControl",
+    base::FEATURE_ENABLED_BY_DEFAULT };
 
-bool IsPageProbabilitiesStudyActive() {
-  if (!base::FieldTrialList::Find(kPageProbabilitiesStudyName)) {
+bool IsPageClassifierEnabled() {
+  return base::FeatureList::IsEnabled(kPageClassifier);
+}
+
+int GetPageProbabilitiesHistorySize() {
+  return GetFieldTrialParamByFeatureAsInt(kPageClassifier,
+      "page_probabilities_history_size", kDefaultPageProbabilityHistorySize);
+}
+
+bool HasActiveStudy() {
+  if (!base::FieldTrialList::Find(kActiveStudyName)) {
     return false;
   }
 
   return true;
 }
 
-std::string GetPageProbabilitiesStudy() {
-  return kPageProbabilitiesStudyName;
+std::string GetStudy() {
+  return kActiveStudyName;
 }
 
-std::string GetPageProbabilitiesFieldTrialGroup() {
+std::string GetGroup() {
   std::string group_name;
-  base::FieldTrial* field_trial = base::FieldTrialList::Find(
-      kPageProbabilitiesStudyName);
+  base::FieldTrial* field_trial = base::FieldTrialList::Find(kActiveStudyName);
   if (!field_trial) {
     return group_name;
   }
 
-  group_name = field_trial->group_name();
   return group_name;
 }
 
-int GetPageProbabilitiesHistorySize() {
-  return GetFieldTrialParamByFeatureAsInt(
-      kContextualAdsControl,
-      "page_probabilities_history_size",
-      kDefaultHistorySize);
-}
-
-void LogPageProbabilitiesStudy() {
-  std::string log;
-  if (!IsPageProbabilitiesStudyActive()) {
-    BLOG(1, "No active experiment");
+void Log() {
+  if (!HasActiveStudy()) {
+    BLOG(1, "No active study found");
   } else {
-    const std::string study = GetPageProbabilitiesStudy();
-    const std::string group = GetPageProbabilitiesFieldTrialGroup();
-    const std::string history_size =
-        base::NumberToString(GetPageProbabilitiesHistorySize());
+    const std::string study = GetStudy();
+    const std::string group = GetGroup();
+    BLOG(1, "Running study " << study << " with group " << group);
+  }
 
-    BLOG(1, "Running active experiment:\n"
-      << "  name: " << study << "\n"
-      << "  group: " << group << "\n"
-      << "  value: " << history_size);
+  if (IsPageClassifierEnabled()) {
+    int history_size = GetPageProbabilitiesHistorySize();
+    BLOG(1, "Page Classifier enabled with history size of " << history_size);
```

Comment:
should we add the word Study after Page Classifier?

Reply:
this would be the feature for the actual page classifier - independent of any active studies.